ci-operator/step-registry/red-hat-data-services/ci/image-mirror/red-hat-data-services-ci-image-mirror-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: red-hat-data-services-ci-image-mirror
+  steps:
+    env:
+      REGISTRY_ORG: modh
+      REGISTRY_SECRET: red-hat-data-services-push-secret
+    pre:
+    - ref: opendatahub-io-ci-image-mirror
```

Comment:
Can I use a `ref` in a different workflow? I'm using https://github.com/openshift/release/blob/master/ci-operator/step-registry/opendatahub-io/ci/image-mirror/opendatahub-io-ci-image-mirror-ref.yaml

Reply:
I'm doing this to avoid duplicating code (opendatahub-io-ci-image-mirror-commands.sh).